clients/drcachesim/scheduler/scheduler.h

```diff
@@ -408,6 +423,20 @@ template <typename RecordType, typename ReaderType> class scheduler_tmpl_t {
          * more frequent diagnostics.
          */
         int verbosity = 0;
+        /**
+         * Output stream for recording the schedule for later replay.
+         * write_recorded_schedule() must be called when finished to write the
+         * in-memory data out to this stream.
+         */
+        archive_ostream_t *record_ostream = nullptr;
```

Comment:
schedule_record_ostream would be better; same below.

Reply:
Hmm, I feel that "schedule" is kind of implied as this is a scheduler, but willing to rename it.  Done.

Reply:
This makes the Doxygen references too long, violating the vera++ check.  Searching again for possible ways to avoid fully qualifying for Doxygen links.

Reply:
Splitting across the comment line (and still indenting + *) seems to work, in a weird way though generating two links: but good enough.